outputs.tf

```diff
@@ -4,35 +4,60 @@
 
 output "cluster_arn" {
   description = "ARN that identifies the cluster"
-  value       = try(aws_ecs_cluster.this[0].arn, null)
+  value       = module.cluster.arn
 }
 
 output "cluster_id" {
   description = "ID that identifies the cluster"
-  value       = try(aws_ecs_cluster.this[0].id, null)
+  value       = module.cluster.id
 }
 
 output "cluster_name" {
   description = "Name that identifies the cluster"
-  value       = try(aws_ecs_cluster.this[0].name, null)
+  value       = module.cluster.name
 }
 
-################################################################################
-# Cluster Capacity Providers
-################################################################################
+output "cloudwatch_log_group_name" {
+  description = "Name of cloudwatch log group created"
+  value       = module.cluster.cloudwatch_log_group_name
+}
+
+output "cloudwatch_log_group_arn" {
+  description = "Arn of cloudwatch log group created"
+  value       = module.cluster.cloudwatch_log_group_arn
+}
 
 output "cluster_capacity_providers" {
   description = "Map of cluster capacity providers attributes"
-  value = {
-    for k, v in aws_ecs_cluster_capacity_providers.this : v.id => v
-  }
+  value       = module.cluster.cluster_capacity_providers
+}
+
+output "autoscaling_capacity_providers" {
+  description = "Map of autoscaling capacity providers created and their attributes"
+  value       = module.cluster.autoscaling_capacity_providers
+}
+
+output "task_exec_iam_role_name" {
+  description = "Task execution IAM role name"
+  value       = module.cluster.task_exec_iam_role_name
+}
+
+output "task_exec_iam_role_arn" {
+  description = "Task execution IAM role ARN"
+  value       = module.cluster.task_exec_iam_role_arn
+}
+
+output "task_exec_iam_role_unique_id" {
+  description = "Stable and unique string identifying the task execution IAM role"
+  value       = module.cluster.task_exec_iam_role_unique_id
 }
 
 ################################################################################
-# Capacity Provider - Autoscaling Group(s)
+# Service(s)
 ################################################################################
 
-output "autoscaling_capacity_providers" {
-  description = "Map of autoscaling capacity providers created and their attributes"
-  value       = aws_ecs_capacity_provider.this
+output "services" {
+  description = "Map of services created and their attributes"
+  value       = module.service
+  sensitive   = true
```

Comment:
Unfortunately. I really dislike this feature of Terraform... :(

Reply:
Yes! See below (its now removed)